umpost/um2netcdf.py

```diff
@@ -345,9 +342,9 @@ def process(infile, outfile, args):
             stashcode = c.attributes['STASH']
             itemcode = to_item_code(stashcode)
```

Comment:
Would it be worth swapping this out for the new `item_code` attribute here, or is it something for further down the line?

Reply:
Yep, I've fixed it.